ts/components/messages/MessageListCTABar.tsx

```diff
@@ -94,6 +96,13 @@ class MessageListCTABar extends React.PureComponent<Props> {
     }
   }
 
+  private renderEUCovidViewCTA() {
+    if (this.props.message.content.eu_covid_cert === undefined) {
+      return null;
+    }
+    return <ViewEUCovidButton onPress={this.props.onEUCovidCTAPress} />;
```

Comment:
```suggestion
private renderEUCovidViewCTA() {
  return (
      this.props.message.content.eu_covid_cert && (
        <ViewEUCovidButton onPress={this.props.onEUCovidCTAPress} />
      )
  );
}    
```
More concise but same effect, what do you think?

Reply:
Much more better! thx